code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -230,6 +241,7 @@ GLOBAL_LIST_INIT(sandbag_recipes, list (
 
 /obj/item/stack/sheet/mineral/uranium
 	name = "uranium"
+	desc = "Don't keep this stuff in your pocket for too long. Hell, don't keep it anywhere near your person for too long."
```

Comment:
Uranium doesn't send out radpulses

Reply:
I am aware. The coin that already exists has a similar description.